pkg/kubelet/dockershim/docker_stats.go

```diff
@@ -17,16 +17,112 @@ limitations under the License.
 package dockershim
```

Comment:
Split the file into _linux and windows.go

Reply:
I have split the file as you suggested.